### ISSUE TITLE
Make treemap resizeable

### ISSUE THREAD

**Issue by kieferm** [jrsconfitto/VisVirtualHackathon#13]
Would [this](http://stackoverflow.com/a/25978286/2202868) snippet work?

**Comment by kieferm** [comment]
This section of code is always coming up with width = 300, height = 150

```
        // Builds a treemap under the passed element
        BuildTreemap: ($treemapElement) => {
            var width = +$treemapElement.width(),
                height = +$treemapElement.height();

            var myTreemap = treemap()
                .width(width)
                .height(height);
```

**Comment by jrsconfitto** [comment]
Hopefully will be fixed by #32
